src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = 0;
+  const DAILY_RENT = 40;
+  const MEDIUM_TERM = 3;
+  const REG_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const BIG_DISCOUNT = 50;
+
+  for (let i = 0; i < days; i++) {
+    totalCost += DAILY_RENT;
+  }
```

Comment:
Why are you using a loop? You can do this with a simple multiplication operation

Reply:
~~I prefer it that way~~

Proof that I did it myself and not just by copying the solution in the video.